torchrl/data/tensordict/metatensor.py

```diff
@@ -69,8 +69,9 @@ def __init__(
         requires_grad: bool = False,
         _is_shared: Optional[bool] = None,
         _is_memmap: Optional[bool] = None,
+        _is_tensordict: Optional[bool] = None,
```

Comment:
`None` allows us to have the possibility of overriding things or not doing checks.

something along the line of 
```python
            if _is_tensordict is None:
                _is_tensordict = not isinstance(tensor, (MemmapTensor, torch.Tensor))
```

We can say: don't check if it's a tensordict, i'm telling you it is / it isn't. This can be useful if you have something else than MemmapTensor or Tensor in this case.